addons/kafka/templates/componentversion-broker.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentVersion
+metadata:
+  name: kafka-broker
+spec:
+  compatibilityRules:
+    - compDefs:
+        - {{ include "kafka-broker.componentDefName" . }}
+      releases:
+        - 3.3.2-debian-11-r54
+  releases:
+    - name: 3.3.2-debian-11-r54
+      changes:
+      serviceVersion: 3.3.2
+      images:
+        kafka: docker.io/bitnami/kafka:3.3.2-debian-11-r54
```

Comment:
Better not to hardcode the repo registry , you can refer to other addons to make the repo configurable.